vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -253,7 +287,13 @@ bool ChakraRuntime::compare(const facebook::jsi::PropNameID &lhs, const facebook
 }
 
 std::string ChakraRuntime::symbolToString(const facebook::jsi::Symbol &s) {
-  return ToStdString(ToJsString(GetChakraObjectRef(s)));
+  facebook::jsi::Value result = global()
+                                    .getPropertyAsFunction(*this, "eval")
+                                    .call(*this, "(function(sym){ return sym.toString(); })")
```

Comment:
How often do we call this function? This seems like it would be very slow.

Reply:
I plan to refactor it to a function that we preload to the runtime as we do for the host object proxy. Speed-wise it will be not slower than anything else. It is amazing how much work each JS engine API call does. In many cases running a specialized predefined JavaScript function and reducing number of JS API calls would be much faster. 

---
In reply to: [460425939](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460425939) [](ancestors = 460425939)